legend-sdlc-server/src/main/java/org/finos/legend/sdlc/server/gitlab/auth/GitLabTokenManager.java

```diff
@@ -19,14 +19,18 @@
 import org.gitlab4j.api.Constants;
 
 import java.io.Serializable;
+import java.time.LocalDateTime;
 import java.util.Objects;
 
 class GitLabTokenManager implements Serializable
 {
     private static final long serialVersionUID = 4579663645788521787L;
+    private static final long DEFAULT_EXPIRY = 7200;
```

Comment:
Let's make clear that we're talking about seconds here. Also, let's store our refresh threshold as a constant.
```suggestion
    private static final long DEFAULT_EXPIRY_SECS = 7200;
    private static final long DEFAULT_REFRESH_THRESHOLD_SECS = 1800;
```